packages/esm-patient-common-lib/src/workspaces/useWorkspaces.tsx

```diff
@@ -30,10 +33,24 @@ export function useWorkspaces(): WorkspacesInfo {
     }
   }, [workspaces]);
 
-  return {
-    active: workspaces.length > 0,
-    windowState,
-    workspaces,
-    prompt,
-  };
+  useEffect(() => {
```

Comment:
This looks like it's intended to replace the `windowState` memo, but it's still left in?

Reply:
Yes, actually I saw [this](https://github.com/openmrs/openmrs-esm-patient-chart/blob/4201b4aa390ead099b09aaebbc0d891b1fa0c112/packages/esm-patient-common-lib/src/workspaces/useWorkspaceWindowSize.tsx#L44) useEffect as well in the WorkspaceWindowSize Context.